### DISCUSSION TITLE
Регистрация/Авторизация клиентов с помощью API

### DISCUSSION THREAD

**Discussion by failhunterby** [totumonline/totum-ru-issues-and-discussions#390] [Как это сделать?] [answered]
Хочу сделать небольшой личный кабинет для клиентов. Необходимо создавать-авторизовывать пользователей в тотум. С документацией не очень понял - возможно это или нет. спасибо!

**Comment by ttm-alexeypolunin** [comment, marked as answer]
Да, возможно. Если вы хотите, что бы пользователи сами регистрировались — сделайте Быструю форму, которая будет добавлять пользователя в таблицу Users

быстрые формы: https://ru.docs.totum.online/forms#q-forms
